src/server/game/Entities/Player/Player.cpp

```diff
@@ -15139,33 +15139,24 @@ void Player::SendPreparedQuest(WorldObject* source)
         // Auto open
         if (Quest const* quest = sObjectMgr->GetQuestTemplate(questId))
         {
-            if (qmi0.QuestIcon == 4)
-            {
+            if (qmi0.QuestIcon == GOSSIP_ICON_INTERACT_1)
```

Comment:
I would like you to revert this change - GOSSIP_ICON_INTERACT_1 is not the correct enum to use (there is no defined enum for this field)